socorro/external/es/super_search_fields.py

```diff
@@ -1765,10 +1765,12 @@ def get(self):
         'default_value': None,
         'description': (
             'The build architecture. Usually one of: "x86", "amd64" (a.k.a. x86-64), "arm", '
-            '"arm64".\n\nDuplicate of cpu_name, with a better name.'
+            '"arm64".'
         ),
         'form_field_choices': [],
         'has_full_version': False,
+        # NOTE(willkg): this uses "cpu_name" to search on, but that's deprecated;
+        # switch to "cpu_arch" after July 2019
         'in_database_name': 'cpu_name',
```

Comment:
Uh... This is a mistake. This is continuing to populate `processed_crash.cpu_name` in the ES document, but in order to transition in July 2019, we need to also add `processed_crash.cpu_arch`. I'll add that now.